commons/zenoh-config/src/wrappers.rs

```diff
@@ -72,6 +81,13 @@ impl From<ZenohId> for OwnedKeyExpr {
     }
 }
 
+impl From<ZenohId> for [u8; ZenohIdProto::MAX_SIZE] {
```

Comment:
Instead of this `From` it would be better to expose `to_le_bytes` which provides stronger guarantees on the endianess.